markdown/3.x/en/api/ff-checkout-tracking.api.md

```diff
@@ -6,10 +6,17 @@ ___
 | **disable-auto-tracking**&nbsp;(Boolean) (default: false) | Controls whether the element sends a tracking request upon its rendering. |
 | **records**&nbsp;(Array) | The records returned from FACT-Finder. |
 
+### Methods
+| Name | Description |
+| ---- | ----------- |
+| **trackCheckoutItems** | Dispatches a tracking checkout event for each of its `ff-checkout-tracking-item` children. |
```

Comment:
The same as here - either `for all children` or `for each child`.

Reply:
Looks fine to me.
Which part exactly is it that you find worth adjusting?

Depending on the component's behaviour, `for all children` might be unsuitable. Only if it is one event for any amount of children (1 to n)

Reply:
I think `each` does not go with `children`. We can say "all children" or "each child". According to [this site](https://www.woodwardenglish.com/all-vs-every-vs-each/) 'all' is followed by plural noun, but 'each' and 'every' - be a singular one.

Reply:
It is indeed true that _"each children"_ doesn't work. In this case, however, we have _"each **of its** children"_ which is correct again.
It corresponds to the _**Each of + You/Us/Them**_ example.

Reply:
I don't want to be a nit-picker here, but I'm really not convinced. I don't say I'm right, but I think **Each of + You/Us/Them** works only for **You/Us/Them**. However, language can be tricky sometimes. Even Polish native speakers make common mistakes when it comes to our language. Would you mind if I ask one of our English teachers? She's a language-freak and will explain it in details.

Reply:
`them` is a pronoun which means you can use it interchangably with what it refers to. `its children` in this case.
Making `them` == `its children`
Therefore, `each of them` == `each of its children`

Here are some more examples. It even lists `Each of the children received a special gift.`
https://dictionary.cambridge.org/grammar/british-grammar/each-or-every

But feel free to ask your English teacher. An additional opinion might help clarifying.

Reply:
The answer was: "you're both right". I think I should treat "of" as a cast operator here that does the trick, as you said. It was just a non-intuitive case for me, but a good change to learn something new. Thank you for your involvement in explaining it to me with details..